INSTALL.md

```diff
@@ -26,33 +25,41 @@ source code.  Unpack as follows.
     tar xzf <some dir>/accumulo-X.Y.Z-bin.tar.gz
     cd accumulo-X.Y.Z
 
+There are four scripts in the the `bin/` directory that are use to manage Accumulo:
+
+1. `accumulo` - Runs Accumulo command-line tools and starts Accumulo processes
+2. `accumulo-service` - Runs Accumulo processes as services
```

Comment:
I like the distinction of start/stop/kill for a service on the given node (I really liked start-daemon.sh in recent versions).